src/MainGrid.js

```diff
@@ -150,69 +157,90 @@ MainGrid.prototype.render = function () {
     _self.svg.on('mouseover', function (d) {
         var target = d3.event.target;
         var coord = d3.mouse(target);
+
         var xIndex = _self.rangeToDomain(_self.x, coord[0]);
         var yIndex = _self.rangeToDomain(_self.y, coord[1]);
-        var obs = _self.observations[target.dataset.obsIndex];
 
-        if (!obs || _self.crosshair) return;
+        if (!target.dataset.obsIndex || _self.crosshair) return;
+        var obsIds = target.dataset.obsIndex.split(' ')
```

Comment:
whats with the split?

Reply:
iirc my reasoning was to do the same check that !obs was doing before,
var obs = _self.observations[target.dataset.obsIndex];
but by finding the donor/gene combo in the observations list. Mutations and cnv targets could have the same data-obs-index so they can share the same mouseover event.